src/policy/fees.h

```diff
@@ -186,6 +186,10 @@ class CBlockPolicyEstimator
     static constexpr double FEE_SPACING = 1.05;
 
     const fs::path m_estimation_filepath;
+
+    /** Fee estimate.dat must not be more than 12 hours old. */
+    static constexpr std::chrono::hours MAX_FILE_AGE{12};
```

Comment:
Yeah, to enhance flexibility, I can create a follow-up PR to introduce a configuration variable for MAX_FILE_AGE,  This way users can specify the maximum age of estimates they do not want to accept?

Reply:
I was thinking that introducing flexibility would enable users to define their preferred threshold for considering fee estimates as stale. As @instagibbs suggested extending the default 12 hours further. what is your view on it?

Reply:
Another use case could be so that testers can purposefully load (very) old fee estimate files to populate `estimatesmartfee`? https://github.com/bitcoin/bitcoin/issues/27415#issuecomment-1531711091

Reply:
Not sure if we should load old (i.e. potentially bad) data as it can skew the fee estimates both at startup and after more data has been collected. For testing, they could mock the time right? Just worried about adding a config option that's a bit footgunny.

Reply:
Yes, I agree with you. stale estimates should not be read. I have updated the MAX_FILE_AGE to 60 hours, which 2.5 days, as @instagibbs suggested that longer would be okay?

Reply:
Can you explain the rationale behind the 60 hours chosen? Also perhaps you can update OP as it still mentions 12 hours :)